projects/observability/src/shared/dashboard/widgets/top-n/data/top-n-data-source.model.ts

```diff
@@ -44,32 +35,50 @@ export class TopNDataSourceModel extends GraphQlDataSourceModel<TopNWidgetDataFe
   })
   public resultLimit: number = 10;
 
+  @ModelProperty({
+    type: ARRAY_PROPERTY.type,
+    key: 'select-option-metrics',
```

Comment:
Since we've moved this and it's a breaking change to the json anyway, what are your thoughts about adding in a rename to just `options`?